ckanext/pdfview/theme/public/vendor/pdfjs/web/compatibility.js

```diff
@@ -24,6 +24,20 @@ if (typeof PDFJS === 'undefined') {
   (typeof window !== 'undefined' ? window : this).PDFJS = {};
 }
 
+// Disable PDFJS Worker due to compatability issues associated w/ redirects & CORS
+// This impacts only Internet Explorer related browswers
+function pdfJSWorkerCompatabilityCheck() {
+    var ua = window.navigator.userAgent;
```

Comment:
There should be a check in case navigator is undefined.
https://github.com/mozilla/pdf.js/blob/version-1.10/src/shared/compatibility.js#L26